test-k8s.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
```

Comment:
We should stick to use one of the Shebang lines. In some files such as `run-docker.sh` Shebang line is specified as `#!/usr/bin/env bash`. I would recommend go for `#!/usr/bin/bash`.

Reply:
See this [article](https://www.baeldung.com/linux/bash-shebang-lines) as well

Reply:
Converted all to `#!/bin/bash`

Reply:
Ok